test/e2e/coverage_docs/scheduling_test.md

```diff
@@ -0,0 +1,26 @@
+| Test schedule the Job with label and group constraints        | Job propagation with label and group constraints testing           |                                                                                                                                |
+
+#### Replica Scheduling Strategy testing
+| Test Case                                                                                                                                                                                    | E2E Describe Text                                       | Comments |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|----------|
+| Test ReplicaScheduling when ReplicaSchedulingType value is Duplicated                                                                                                                        | replicas duplicated testing                             |          |
+| Test ReplicaScheduling when ReplicaSchedulingType value is Duplicated, trigger rescheduling when replicas have changed                                                                       | replicas duplicated testing when rescheduling           |          |
+| Test ReplicaScheduling when ReplicaSchedulingType value is Divided, ReplicaDivisionPreference value is Weighted, WeightPreference is nil                                                     | replicas divided and weighted testing                   |          |
+| Test ReplicaScheduling when ReplicaSchedulingType value is Divided, ReplicaDivisionPreference value is Weighted, WeightPreference is nil, trigger rescheduling when replicas have changed    | replicas divided and weighted testing when rescheduling |          |
+| Test ReplicaScheduling when ReplicaSchedulingType value is Divided, ReplicaDivisionPreference value is Weighted, WeightPreference isn't nil                                                  | replicas divided and weighted testing                   |          |
+| Test ReplicaScheduling when ReplicaSchedulingType value is Divided, ReplicaDivisionPreference value is Weighted, WeightPreference isn't nil, trigger rescheduling when replicas have changed | replicas divided and weighted testing when rescheduling |          |
+
+#### Job Replica Scheduling Strategy testing
+| Test Case                                                     | E2E Describe Text                         | Comments |
+|---------------------------------------------------------------|-------------------------------------------|----------|
+| Test Job's parallelism are divided equally on member clusters | Job replicas divided and weighted testing |          |
+
+#### TODO
+1. Update the `fmt.Printf` to `klog.Infof`
```

Comment:
What does this todo mean?

Reply:
i think we should use `klog.info` in the whole test file? im not sure of that, there are some `fmt.Printf` i think it should update to `klog.infof`

Reply:
Based on the test results, it can be seen that using `fmt.Printf` can output correct logs and facilitate troubleshooting.

Reply:
ok, i will remove this TODO~